### PR TITLE
Update de.po - Filmic as a trademark

### PR DESCRIPTION
**Pull request by MStraeten**
using "Filmic" as a trademark instead of "Filmisch" which means rather cinematic
some correction regarding profiles
@bilddateien, @da-phil, @rabauke  please review

### CODE REVIEW

**Review by TurboGit — APPROVED** [review]
Thanks !

File: po/de.po [hunk 1]
```diff
@@ -19320,68 +19486,111 @@ msgstr "neue Sitzung „%s” gestartet"
...
+#~ "Lade keine Dateien mit der Endung .jpg oder .jpeg. Dies kann nützlich "
+#~ "sein, wenn es Raw+JPEG im gleichen Ordner gibt."
+
+#~ msgctxt "accel"
+#~ msgid "leave fullscreen"
+#~ msgstr "Vollbild verlassen"
+
+# zu frei?
+#~ msgid "import options"
+#~ msgstr "Import-Optionen"
```
Comment (da-phil) [inline po/de.po:19583]: Ne, passt ;)

**Comment by da-phil** [comment]
> using "Filmic" as a trademark instead of "Filmisch" which means rather cinematic
> some correction regarding profiles
> @bilddateien, @da-phil, @rabauke please review

Fine for me too :+1:

**Comment by rabauke** [comment]
This commit/merge is ok for me. I have a small personal preference for "Filmkurve" or possibly "Filmic-Kurve" over "Filmic". As mentioned before, not translating "Filmic" at all and using it in a trademark-like fashion is much more appropriate than the verbatim translation "filmisch".

**Pull request merged**